amanuensis/resources/admin/admin_associated_user.py

```diff
@@ -29,6 +30,16 @@ def get_codes_for_roles():
         roles = udm.get_associated_user_roles(session)
         return {data.role: data.code for data in roles}
 
+def update_associated_user_user_id(logged_user_id, logged_user_email):
+    with flask.current_app.db.session as session:
+        user = udm.associate_user.get_associated_user(session, logged_user_email)
+        if not user:
+            raise NotFound("Associated User does not exist") 
+        if not user.user_id:
+            user.user_id = logged_user_id
+            return udm.associate_user.update_associated_user(session, user)
+        else:
+            return "200"
```

Comment:
just return, Instead of returning 200 (which would be on the blueprint / router level you should return the updated entity here. I guess this is true also for the other 2 returns in this function.

Reply:
I just returned the user object in case there is a future use where that is needed. I can remove that if you just want it to return None at the end